Added structure and conf.py changes

Souptacular opened the pull request:
set-up structure and made conf.py changes for Solidity syntax highlighting.

Still need to create the rest of the .rst pages and obtain .md pages from frontier guide (they aren't available publicly afaik).

zelig commented on source/introduction.rst:
```diff
@@ -0,0 +1,13 @@
+
+   what-is-ethereum.rst
+   the-homestead-release.rst
+   solidity-in-depth.rst
```
this should not be here right?

Souptacular replied:
I don't see why not. the toctree is a common thing for the the larger sub-pages of a guide. It won't be the *only* thing necessarily, but I was putting it there at first so there wouldn't be a blank page.

zelig replied:
no i meant solidity-in-depth is not supposed to be in the intro

zelig commented on source/index.rst:
```diff
@@ -20,3 +31,107 @@ Indices and tables
+    * [Table of Sub Units]
+    * Getting Ether [Will point to mining and exchanges pages with explanation of both]
+    * Sending Ether
+  - Mining
```
so here we need a section on running a node, syncing, fast syncing and light client
network connectivity , NAT, discovery and peers

zelig commented on source/index.rst:
```diff
@@ -20,3 +31,107 @@ Indices and tables
+    * IDEs/Frameworks
+    * High Level Languages
+    * Other Tools
+  - DApp Development
```
solidity resources should come here right

Souptacular commented:
Fixed issues and added structure of documents and placeholder pages/sections for the documentation. Can be viewed on https://ethereum-homestead.readthedocs.org/en/latest/ or built locally (I fixed the issue preventing local builds).

zelig commented:
fantastic thx

zelig commented:
merged :+1:
